dev-packages/cli/src/download-plugins.ts

```diff
@@ -195,6 +195,16 @@ export default async function downloadPlugins(options: DownloadPluginsOptions =
     }
 }
 
+const placeholders: { name: string, replaceValue: string }[] = [
+    { name: 'targetPlatform', replaceValue: process.platform + '-' + process.arch }
+];
```

Comment:
Suggestion: Using a simply record here might be preferable. That requires to modify the `resolveDownloadUrlPlaceholders` function as well.

```suggestion
const placeholders: Record<string, string> = {
    targetPlatform: process.platform + '-' + process.arch
};
```